### ISSUE TITLE
Different select code for LG TV

### ISSUE THREAD

**Issue by makeworld-the-better-one** [Lucaslhm/Flipper-IRDB#12]
Currently the code for the Select button for LG C1 is this: https://github.com/Lucaslhm/Flipper-IRDB/blob/308cd339da9d2c0195ad212bfa15ca7f75328607/TVs/LG_C1.ir#L58-L62

All the buttons in that file worked with my LG TV, except the Select button. I recorded my Select button and got this code:

```
name: Select
type: parsed
protocol: NEC
address: 04 00 00 00
command: 44 00 00 00
```

I don't have an LG C1, but all the other buttons worked... maybe the LG C1 would respond to my Select button as well? If someone could test that'd be great, and then I could make a PR swapping mine in. Otherwise I'll make a separate file for my model, the LG 42LF5800 (Smart LED TV).

**Comment by UberGuidoZ** [comment]
Added downstream:
 - https://github.com/logickworkshop/Flipper-IRDB/compare/main...UberGuidoZ:main

Now part of pending pull request:
 - https://github.com/Lucaslhm/Flipper-IRDB/pull/31

**Comment by UberGuidoZ** [comment]
Merged as part of #31 above. Thank you! Closing.

**Comment by jjfeldcher** [comment]
The file was not updated, but the code is correct.
